R/sf.R

```diff
@@ -61,6 +61,10 @@
 #' @name ggsf
 NULL
 
+geom_column = function(data) {
+  data[[ which(sapply(data, function(x) inherits(x, "sfc")))[1] ]]
```

Comment:
Can you please use `vapply()` here instead of `sapply()`? You should be able to drop the `function()` too.

(And `<-` instead of `=`)